tests/unit/h/views/activity_test.py

```diff
@@ -1189,7 +1189,7 @@ def user_search_controller(self, user, pyramid_request):
 @pytest.fixture
 def group(factories):
     group = factories.Group()
-    group.members.extend([factories.User(), factories.User()])
+    group.members.extend([group.creator, factories.User(), factories.User()])
```

Comment:
No tests actually fails if I _don't_ add the creator as a member in this fixture but it's hard to know whether I've broken anything but it's just still passing. This is a massive test file. Thought it best to just maintain the previous behavior of this fixture.